docs/user-guide/consolidated_metadata.rst

```diff
@@ -1,7 +1,7 @@
 Consolidated Metadata
 =====================
 
-Zarr-Python implements the `Consolidated Metadata_` extension to the Zarr Spec.
+Zarr-Python implements the `Consolidated Metadata`_ extension to the Zarr Spec.
```

Comment:
If I search fo "consolidated" on this link it doesn't turn up with anything - can this be fixed to link to a more suitable place than the zarr spec itself, which doesn't mention consolidated metadata?